Library/Homebrew/test/cask/cli/quarantine_spec.rb

```diff
@@ -0,0 +1,91 @@
+describe Hbc::Quarantine, :cask do
+  describe "by default" do
+    it "quarantines Cask installs" do
+      Hbc::CLI::Install.run("local-transmission")
+
+      expect(Hbc::CaskLoader.load(cask_path("local-transmission"))).to be_installed
+      expect(Hbc::Config.global.appdir.join("Transmission.app")).to be_a_directory
+
+      expect(
+        described_class.detect(Hbc::Config.global.appdir.join("Transmission.app")),
```

Comment:
Add a custom `be_quarantined` matcher for this. Here's an example:

https://github.com/Homebrew/brew/blob/459e2bfa48ebed4bf49d6aebb68df1b416e9b0f3/Library/Homebrew/test/missing_formula_spec.rb#L11-L15